library/standard_tables_capture.inc

```diff
@@ -327,7 +327,7 @@ function icd_import($type) {
     if( is_dir($dir) && $handle = opendir($dir)) {
         while (false !== ($filename = readdir($handle))) {
 	    // bypass unwanted entries
-	    if (!stripos($filename, ".txt") || stripos($filename,"diff")) {
+	    if (!stripos($filename, ".txt") || stripos($filename,"diff") || stripos($filename, "addenda")) {
```

Comment:
here's a snippet, i don't think we need to program for this but i'll test in an upgrade to see if it makes a difference

Add:         1 C49A9   Gastrointestinal stromal tumor of other sites                Gastrointestinal stromal tumor of other sites
Revise from: 1 C7A094  Malignant carcinoid tumor of the foregut NOS                 Malignant carcinoid tumor of the foregut NOS
Revise to:   1 C7A094  Malignant carcinoid tumor of the foregut, unspecified        Malignant carcinoid tumor of the foregut, unspecified